zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/Learner.java

```diff
@@ -246,18 +246,23 @@ void request(Request request) throws IOException {
             LOG.error("Throttled request sent to leader: {}. Exiting", request);
             ServiceUtils.requestSystemExit(ExitCode.UNEXPECTED_ERROR.getValue());
         }
-        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        // size of sessionId, cxId and type in bytes
+        int size = Long.BYTES + 2 * Integer.BYTES;
+        byte[] bytes = null;
+        if(request.request != null) {
```

Comment:
you need a space after the 'if' statement to fix the checkstyle error.
(btw you can run checkstyle locally by: `mvn verify spotbugs:check checkstyle:check -DskipTests` )

Reply:
Oh man thanks a lot. I thought I fixed that one but it turns out there is another one out there :(